### PR TITLE
Addresses #3086 NTR pulmonary interlobular septum

### PR DESCRIPTION
**Pull request by aleixpuigb**
Addresses #3086 NTR pulmonary interlobular septum

### CODE REVIEW

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Review by bvarner-ebi — COMMENTED** [review]
See inline comments.

Will approve if requester and/or SME confirm no changes are desired in response to comments.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224538,6 +224538,22 @@ intersection_of: part_of UBERON:0004027 ! chorionic plate
...
+synonym: "fibrous septum" RELATED [PMID:3259815]
+synonym: "interlobular interstitium" RELATED [doi:10.53347/rID-8764]
+synonym: "interlobular septa" BROAD PLURAL [PMID:3259815]
+synonym: "interlobular septum" BROAD [doi:10.53347/rID-8764]
+is_a: UBERON:0000114 ! lung connective tissue
+is_a: UBERON:0003037 ! septum
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+relationship: overlaps UBERON:0002016 ! pulmonary vein
+relationship: overlaps UBERON:0018227 ! pulmonary lymphatic vessel
+relationship: surrounds UBERON:0010369 ! secondary pulmonary lobule
```
[the remark this answers is not in the document]
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224554]: Thank you for your comment. The definition of surrounds is "the inverse of surreounded by", which is:
> x surrounded_by y if and only if (1) x is adjacent to y and for every region r that is adjacent to x, r overlaps y (2) the shared boundary between x and y occupies the majority of the outermost boundary of x

Although the `pulmonary interlobular septum` has some micropores, it occupies the majority of the outermost surface around the `secondary pulmonary lobule`.

From PMID:3259815: "Surrounding the secondary lobule are interlobular septa containing venules and lymphatics"
Supported with the image:
image

I can add this [reference](https://www.researchgate.net/figure/Drawing-of-the-secondary-pulmonary-lobule-of-the-lung-The-interlobular-septa-runs-along_fig2_346517053) from DOI:[10.1016/j.ejro.2020.100274](http://dx.doi.org/10.1016/j.ejro.2020.100274) if it makes it more clear.

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224538,6 +224538,22 @@ intersection_of: part_of UBERON:0004027 ! chorionic plate
 relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
 property_value: dcterms-date "2023-09-14T09:09:36Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600022
+name: pulmonary interlobular septum
+def: "A septum that surrounds and gives shape to the secondary pulmonary lobules, it is composed of connective tissue within which run the pulmonary venules and lymphatic vessels." [doi:10.53347/rID-8764, PMID:3259815]
+synonym: "fibrous septum" RELATED [PMID:3259815]
```
[the remark this answers is not in the document]
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224545]: Agree is descriptive, I will remove it.

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -224538,6 +224538,22 @@ intersection_of: part_of UBERON:0004027 ! chorionic plate
...
+synonym: "interlobular interstitium" RELATED [doi:10.53347/rID-8764]
+synonym: "interlobular septa" BROAD PLURAL [PMID:3259815]
+synonym: "interlobular septum" BROAD [doi:10.53347/rID-8764]
+is_a: UBERON:0000114 ! lung connective tissue
+is_a: UBERON:0003037 ! septum
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+relationship: overlaps UBERON:0002016 ! pulmonary vein
```
[the remark this answers is not in the document]
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224552]: - As one reference referred to venules and the other to veins, I have looked for an additional reference that also states that venules go through the septa. I will Add `pulmonary venule` as a new term, and also add this reference. PMID:[22283705](https://pubmed.ncbi.nlm.nih.gov/22283705).

- Which object property would you recommend?

[the remark this answers is not in the document]
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:224552]: I will remove the pulmonary vein axiom, and in ticket #3096 I will add:
'pulmonary interlobular septum' SubclassOf overlaps some 'pulmonary venule'.

[the remark this answers is not in the document]
Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo]: > Which object property would you recommend?

Not sure if there is a standard in how to relate vessels passing through a structure or if all septa would "overlap" with these vessels. This is not a breaking edit, and defer to SME's guidance.

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**